File: Julia/keygen.jl
cd_key = Array{String}(undef, 2)
chars = ["2", "3", "7", "a", "b", "c", "d", "g", "h", "j", "l", "p", "r", "s", "t", "w"]

for i in 0:15
	push!(cd_key, chars[rand(1:16)])
end
cd_key_print = join(cd_key,',')
println(cd_key_print)

File: Ruby/keygen.rb
cd_key = Array.new
chars = ['2', '3', '7', 'a', 'b', 'c', 'd', 'g', 'h', 'j', 'l', 'p', 'r', 's', 't', 'w']

for i in 0..16
  cd_key.push(chars[rand(1..16)])
end
puts(cd_key.join)

# or if you want the riffle-shuffle one liner

puts(['2', '3', '7', 'a', 'b', 'c', 'd', 'g', 'h', 'j', 'l', 'p', 'r', 's', 't', 'w'].shuffle.join)
